src/qt/editaddressdialog.cpp

```diff
@@ -164,3 +167,22 @@ void EditAddressDialog::setAddress(const QString &_address)
     this->address = _address;
     ui->addressEdit->setText(_address);
 }
+
+// Address changed
+void EditAddressDialog::addressEdited(const QString& text)
```

Comment:
Perhaps name the argument like in `setAddress()` just above.
```suggestion
void EditAddressDialog::addressEdited(const QString& address)
```